.github/actions/setup-playwright/action.yml

```diff
@@ -5,13 +5,13 @@ inputs:
   node-version:
     description: 'Node.js version to use'
     required: false
-    default: '22'
+    default: '25'
```

Comment:
Have you verified the combination of node/Playwright runs fine locally? Last time we did the same, we faced some troubles.

Reply:
I saw [you launched it remotely](https://chat.kaleidos.net/penpotqa/pl/e681yot4d7fdudhosoudxmntkw). Perfect!

Reply:
yess :)